File: 2763/2763.go
func sumOfImbalanceNumbersOfAllSuba(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := sumOfImbalanceNumbersOfAllSuba(root.Left)
	r := sumOfImbalanceNumbersOfAllSuba(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 2763/2763.cs
public class Solution {
    public int SumOfImbalanceNumbersOfAllSuba(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(SumOfImbalanceNumbersOfAllSuba(root.left), SumOfImbalanceNumbersOfAllSuba(root.right));
    }
}